README.adoc

```diff
@@ -16,14 +16,12 @@ You will build a microservice application that uses Spring Cloud LoadBalancer to
 client-side load-balancing in calls to another microservice.
 
 
-== What You Will Need
-
-* About 15 minutes
-* A favorite text editor or IDE
-* JDK 1.8 or later
-* Gradle 6+ or Maven 3.5+
-* You can also import the code straight into your IDE:
-* Spring Tool Suite (STS) or IntelliJ IDEA
+== What You Need
```

Comment:
We don't want to lose the `About 15 minutes` line. That's a marketing point for the guides.